packages/suite-web/e2e/tests/analytics/toggle.test.ts

```diff
@@ -24,6 +24,7 @@ describe('Analytics Toggle - Enablement and Disablement', () => {
         cy.interceptDataTrezorIo(requests).as('data-fetch');
 
         cy.prefixedVisit('/');
+        cy.disableFirmwareHashCheck();
```

Comment:
Similarly to the previous file, we could refactor 
cy.interceptDataTrezorIo(requests).as('data-fetch');
cy.prefixedVisit('/');
cy.disableFirmwareHashCheck();

to beforeEach if I am not mistaken

Reply:
fixed in 9b8bd8bdd2d014b3d78dc61d1ff0d0e79e5dec86